### DISCUSSION TITLE
Can thresholds be specified as a variable or query?

### DISCUSSION THREAD

**Discussion by dayeondev** [grafana/grafana#53189] [Dashboards, Panels & Visualizations]
Hello, I'm a beginner at grafana. 
I want to update the thresholds of the gauge panel automatically.

There is a relevant issue here.
[Support of variable thresholds #29497](https://github.com/grafana/grafana/issues/29497)
Looking at the bottom [comment](https://github.com/grafana/grafana/issues/29497#issuecomment-880525300) of the issue page above, it seems that the problem has been solved in v8.1. However, the link is currently 404.

I think this function is necessary even now. Also, I wonder how to solve it and why it disappeared.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
